roles/uperf-scale/tasks/wait_client_ready.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- block: 
+  ### <POD> kind
+
+  - name: Get client pod status
+    k8s_facts:
+      kind: Pod
+      api_version: v1
+      namespace: '{{ operator_namespace }}'
+      label_selectors:
+        - app = uperf-scale-bench-client-{{ trunc_uuid }}
```

Comment:
I remember running into 63 chars limitation with k8s name. Would you have enough left for trunc_uuid

Reply:
Yes. verified that.